assets/src/edit-story/components/panels/design/pageAttachment/pageAttachment.js

```diff
@@ -108,16 +107,39 @@ function PageAttachmentPanel() {
   );
 
   const isDefault = _ctaText === defaultCTA;
+  const showTextInput = Boolean(url) && !isInvalidUrl;
+
+  const handleChange = useCallback(
+    ({ target }) => {
+      const { value } = target;
+      // This allows smooth input value change without any lag.
+      _setCtaText(value);
+      debouncedCTAUpdate(value);
+    },
+    [debouncedCTAUpdate]
+  );
+
+  const handleBlur = useCallback(
```

Comment:
I think we should check for trim() on blur to block people like me trying to break it with 5 spaces?

![Kapture 2021-03-04 at 14 37 24](https://user-images.githubusercontent.com/10720454/110033864-2b4a7b80-7cf7-11eb-9c17-aafae5c72d8b.gif)

![Kapture 2021-03-04 at 14 38 50](https://user-images.githubusercontent.com/10720454/110034179-664caf00-7cf7-11eb-9334-6c29cb591152.gif)